service/build.gradle

```diff
@@ -50,74 +51,88 @@ repositories {
-    implementation group: "com.google.cloud", name: "google-cloud-bigquery"
-    implementation group: "com.google.cloud", name: "google-cloud-storage"
-    implementation group: "com.google.guava", name: "guava"
-    implementation group: "com.google.apis", name: "google-api-services-oauth2", version: "v2-rev20200213-2.0.0"
-
-    // OpenApi/Swagger
-    implementation group: "com.fasterxml.jackson.dataformat", name: "jackson-dataformat-yaml", version: "2.11.2"
-    implementation group: "com.fasterxml.jackson.module", name: "jackson-module-jaxb-annotations", version: "2.11.2"
-    implementation group: "com.fasterxml.jackson.core", name: "jackson-core", version: "2.12.3"
-    implementation group: "com.fasterxml.jackson.datatype", name: "jackson-datatype-jsr310", version: "2.12.3"
-    implementation group: "io.swagger.core.v3", name: "swagger-annotations"
-    implementation group: "io.swagger", name: "swagger-annotations", version: "1.6.2"
-    implementation group: "org.webjars", name: "webjars-locator-core", version: "0.48"
-    runtimeOnly group: "org.webjars.npm", name: "swagger-ui-dist", version: "3.36.2"
-    swaggerCodegen group: "io.swagger.codegen.v3", name: "swagger-codegen-cli"
+    implementation "org.apache.commons:commons-text:${gradle.vApacheCommonsText}"
+    implementation 'org.apache.commons:commons-dbcp2'
+    implementation 'org.apache.commons:commons-pool2'
 
     // Spring version controlled dependencies
-    annotationProcessor group: "org.springframework.boot", name: "spring-boot-configuration-processor"
-    implementation group: "javax.validation", name: "validation-api"
-    implementation group: "org.apache.commons", name: "commons-dbcp2"
-    implementation group: "org.apache.commons", name: "commons-pool2"
-    implementation group: "org.springframework.boot", name: "spring-boot-starter-web"
-    implementation group: "org.springframework.boot", name: "spring-boot-starter-data-jdbc"
-    implementation group: "org.springframework.security", name: "spring-security-core"
-
-    // Spring version controlled JUnit dependencies.
+    annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jdbc'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.security:spring-security-core'
+    implementation 'jakarta.validation:jakarta.validation-api'
+    implementation 'jakarta.servlet:jakarta.servlet-api:6.0.0'
+    implementation 'jakarta.ws.rs:jakarta.ws.rs-api:4.0.0'
+
+    // [Application DB] MySQL/MariaDB
+    implementation 'org.mariadb.jdbc:mariadb-java-client:3.4.0'
+    implementation 'com.mysql:mysql-connector-j:8.4.0'
+    implementation 'com.google.cloud.sql:mysql-socket-factory-connector-j-8:1.18.1'
+
+    // GCP libraries. versions are controlled by the BOM specified in buildSrc.
```

Comment:
nit: "Most versions" because the oauth2 library is not?

Reply:
This not controlled in the bom and version needs to be specified, moved to separate block